examples/worlds/performer_detector.sdf

```diff
@@ -0,0 +1,494 @@
+<?xml version="1.0" ?>
+
+<!--
+  Try sending commands:
+
+    To move the vehicle:
+
+      ign topic -t "/model/vehicle_blue/cmd_vel" -m ignition.msgs.Twist -p "linear: {x: 0.5}"
```

Comment:
How about also letting them know to echo the detector topic?

`ign topic -e -t /performer_detector`

Reply:
https://github.com/ignitionrobotics/ign-gazebo/pull/125/commits/728969a2fc588a2cf834988ddf456e7618a298f8